test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperOutputTest.cs

```diff
@@ -79,13 +79,14 @@ public void GenerateStartTag_ReturnsFullStartTag()
                 {
                     { "class", "btn" },
                     { "something", "   spaced    " }
-                });
+                },
+                htmlEncoder: new TestHtmlEncoder());
```

Comment:
I don't think we should be touching existing tests and providing odd encoders. The default encoder is fine.